README.md

```diff
@@ -110,23 +98,94 @@ This project is licensed under the AGPL-3.0 License - see the [LICENSE.md](LICEN
 
 ## Commands
 
-Run dev server: `npm run serve`;<br/>
-Update contrib: `npm run infra:latest`;<br/>
-Bump version: `npm run deploy:prepare`;<br/>
+Run dev server: `yarn dev`;<br/>
+Bump version: `yarn release`;<br/>
 
 <a name="configuration"></a>
 ## Configuration
 
-Plugin configuration:<br/> > `expandOnFirstPlay`: boolean - if plugin should automatically opens on first play (default true);<br/> > `forceChaptersThumb`: boolean - force to use chapters thumbnails (default false);<br/> > `expandMode`: string - expand mode of kitchensink (AlongSideTheVideo|OverTheVideo, default "AlongSideTheVideo");<br/> > `userRole`: string - use session userId as identificator of user (anonymousRole|unmoderatedAdminRole, default "anonymousRole");<br/> > `itemsOrder`: object< string, number > - define order of Tabs (min value at the left) and Items inside group (min value at the top).<br/> itemsOrder also uses as tabs filter (all items that not included in "itemsOrder" object will be filtered out) (default:
-` { "All": 0, "Chapter": 1, "Slide": 2, "Hotspot": 3, "AnswerOnAir": 4, } ); `
+#### Configuration Structure
+
+```js
+//Default configuration
+"navigation" = {};
+//Plugin params
+"navigation" = {
+  expandOnFirstPlay?: boolean, // optional
+  forceChaptersThumb?: boolean, // optional
+  expandMode?: string, // optional
+  userRole?: string, // optional
+  itemsOrder?: object< string, number > // optional
+}
+```
 
-## Troubleshooting
+##
+
+> ### config.expandOnFirstPlay
+>
+> ##### Type: `boolean`
+>
+> ##### Default: `true`
+>
+> ##### Description: if plugin should automatically opens on first play.
+>
+
+##
+
+> ### config.forceChaptersThumb
+>
+> ##### Type: `boolean`
+>
+> ##### Default: `false`
+>
+> ##### Description: force to use chapters thumbnails.
+>
+
+##
+
+> ### config.expandMode
+>
```

Comment:
@MishelYeghiazaryan I think that the expandMode available configuration strings are 'alongside', 'hidden', 'over'. You can verify with @semarche-kaltura

Reply:
enums are:
`
{
  ALONGSIDE: 'alongside', 
  HIDDEN: 'hidden', 
  OVER: 'over'
}
`